stdlib/public/Concurrency/DistributedActor.swift

```diff
@@ -125,6 +159,71 @@ public enum ActorResolved<Act: DistributedActor> {
   case makeProxy
 }
 
+/******************************************************************************/
+/*************************** Actor Personality ********************************/
+/******************************************************************************/
+
+// Personality of distributed actors it means either a 'remote' or 'local'
+// instance of a distributed actor. It is the same actor type, but playing
+// a different role: either a real instance, or a proxy type.
+
+/// A distributed actor's persona represents whether it is a remote proxy
+/// or local instance. The 'local' case contains all the actual state of
+/// the `distributed actor` as it was declared, while the properties on the
+/// outside of the instance are synthesized with replacement accessors
+/// reaching "into" the instance, or failing to do so.
+///
+/// E.g. non distributed properties and functions on a distributed actors will crash the program
+/// when attempted to be read from an actor with a remote persona.
+public enum DistributedActorStorage<Storage> {
```

Comment:
todo rename to `DistributedActorPersonality` remote/local and only local has storage